src/components/dropdown-menu/DropdownMenuSubMenu.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import * as radixDropdownMenu from '@radix-ui/react-dropdown-menu';
+import { forwardRef, ReactNode } from 'react';
+import { IconChevronRight } from '../icons';
+import { A11yAttrs, styleDropdownMenuOverlay } from './DropdownMenuCommons';
+import { DropdownMenuItemBase } from './DropdownMenuItemBase';
+
+interface DropdownMenuSubProps extends radixDropdownMenu.DropdownMenuSubProps {
+  className?: string;
+  id?: string;
+  isOpen?: boolean;
+  isOpenOnMount?: boolean;
+  items: ReactNode | undefined;
+}
+
+export const DropdownMenuSubMenu = forwardRef<HTMLDivElement, DropdownMenuSubProps>(
+  (props, ref) => {
+    const {
+      children,
+      className,
+      id = 'dropdown-submenu',
+      isOpen,
+      isOpenOnMount,
+      items,
+      ...radixProps
+    } = props;
+
+    const a11yAttrs: A11yAttrs = {
+      'aria-controls': `${id}-subdropdown`,
+      id: `${id}-subtrigger`,
+    };
```

Comment:
From what I can see on radix demo component, these arias fields are handled by radix:
![image](https://github.com/user-attachments/assets/4f24831d-8a2e-4280-b136-734aa3560167)
![image](https://github.com/user-attachments/assets/1265f783-14ff-4b92-9ea7-aa8119145346)

Reply:
If we did the same thing in the DropdownMenu base component, then I think we should investigate why we did it there and either drop it or comment on the reason why we have it there.
But radix also attach these props on the main trigger and main dropdown, so I think this is already handled for us and we shouldn't have to worry about it.

Reply:
I probably copied this from the legacy `design-system/` Dropdown - removing it